### PR TITLE
Add info about how to get help to readme

### PR DESCRIPTION
**Pull request by flrs-edf**
Added slack and iea wind task 43 website link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,14 @@ A working group focuses on standardizing measurement data headers. The following
...
 
 We welcome all contributions including issue reporting, new features and bug fixes.   For full details see the contributing document here:
 
 - [Contributing](./contributing.md)
+
+## Getting Help
+
+Please file a new issue in this repository with questions or concerns you might have. If you would like to chat directly with the contributors, please join our Slack channel at [ieawinddigitalization.slack.com](https://ieawinddigitalization.slack.com/).
+
+You can find out more about the IEA's Wind Task 43 working group at [ieawindtask43.org](https://www.ieawindtask43.org/).
```
Comment (stephenholleran) [inline README.md:38]: Should we put the link to the WP4 WRA page?
https://www.ieawindtask43.org/work-package-4-digital-wra

Comment (stephenholleran) [reply README.md:38]: @flrs-edf not sure if you got my comment above?

Comment (kersting) [reply README.md:38]: @stephenholleran Can I go ahead and merge it?

Comment (flrs-edf) [reply README.md]: @stephenholleran I updated the link.

**Review by kersting — APPROVED** [review]
It looks good to me.

**Pull request merged**